docs/index.md

```diff
@@ -1,11 +1,46 @@
 
-`earthaccess` is a python library to **search for**, and **download** or **stream** NASA Earth science data with just a few lines of code.
+# Welcome to `earthaccess`!
 
-Open science only reaches its full potential if we have easy-to-use workflows that facilitate research in an inclusive, efficient and reproducible way. Unfortunately —as it stands today— scientists and students alike face a steep learning curve adapting to systems that have grown too complex and end up spending more time on the technicalities of the tools, cloud and NASA APIs than focusing on their important science.
+`earthaccess` is a Python library for searching, downloading, and streaming NASA Earth science data.
 
-During several workshops organized by [NASA Openscapes](https://nasa-openscapes.github.io/events.html), the need to provide easy-to-use tools to our users became evident. Open science is a collaborative effort; it involves people from different technical backgrounds, and the data analysis to solve the pressing problems we face cannot be limited by the complexity of the underlying systems. Therefore, providing easy access to NASA Earthdata regardless of the data storage location (hosted within or outside of the cloud) is the main motivation behind this Python library.
+Learn more through a recent  [poster](https://docs.google.com/presentation/d/e/2PACX-1vT8BN1C4FkHrid_eVfOwJjZzKGn9Nomb6mnDj3Nld-nD0ekzoenxQIq6Xzo9VFQsXbixCNskadBwPwM/pub#slide=id.g317ff1cef8c_1_87), [slides](https://docs.google.com/presentation/d/1K5RbQj4OKWt49kznIF9ct-cmWADlvdYA0eI7dA7_fFg/edit#slide=id.g269ad4ab477_0_691) and [video](https://www.youtube.com/watch?v=EIr3j1_wDc0)!
 
-The library is an open source community effort under an [MIT license](LICENSE.txt).  We welcome contributions to improve `earthaccess`.  Please see the [Contributing Guide](contributing/index.md) to learn how to get involved.
 
-***earthaccess*** handles authentication with [NASA's Earthdata Login (EDL)](https://urs.earthdata.nasa.gov), search using NASA's [CMR](https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html) and access through [`fsspec`](https://github.com/fsspec/filesystem_spec).
+## Get Started
+
+Using your command line, install `earthaccess` from [PyPI](https://pypi.org) using `pip`:
+
+```
+python -m pip install earthaccess
+```
+
+Or from [conda-forge](https://conda-forge.org) using `mamba` or `conda`:
+
+```
+mamba install earthaccess
+```
+
+## Why `earthaccess`?
+    
+![*NASA Earthdata using `earthaccess` for impactful science. Art by Allison Horst and Openscapes.*](images/earthaccess_environmental_horst.jpg){alt="3-panel watercolor artwork of environmental data within mountains and streams with arrows labeled 'import' and 'earthaccess' that connect to a data analysis panel and then a communicate panel" width="100%"}
+
+<br>
+    
+We can use `earthaccess` to do impactful science, faster:
+
+- Search Data: Explore NASA's extensive collection of Earth science data.
```

Comment:
```suggestion
- Search Data: Explore NASA's extensive collection of Earth science data. Powered by [python-cmr](https://github.com/nasa/python_cmr).
```

Inspired by recent discussions about how python-cmr's importance as an earthaccess component may be under-communicated.